test/e2e/flowvisibility_test.go

```diff
@@ -489,6 +516,96 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 	})
 }
 
+func checkGrafanaQueryResults(t *testing.T, data *TestData, dashboardName, dashboardUid string, queryList *[]query) {
+	queries, err := getQueriesByDashboard("/api/dashboards/uid/", dashboardUid, http.MethodGet, queryList)
+	if err != nil {
+		t.Errorf("Error when getting queries from dashboard: %s\nuid:%s, err: %v", dashboardName, dashboardUid, err)
+		failOnError(err, t, data)
+	}
+	err = checkQueryResult(t, "/api/ds/query", http.MethodPost, queries, queryList)
+	if err != nil {
+		t.Errorf("Error when checking query results from dashboard: %s\nuid:%s, err: %v", dashboardName, dashboardUid, err)
+		failOnError(err, t, data)
+	}
+}
+
+func createNewHttpRequest(httpMethod, url string, body io.Reader) (*http.Request, error) {
+	encodedCreds := base64.StdEncoding.EncodeToString([]byte(grafanaCred))
+	req, err := http.NewRequest(httpMethod, url, body)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Add("Authorization", "Basic "+encodedCreds)
+	req.Header.Add("Accept", "application/json")
+	req.Header.Add("Content-Type", "application/json")
+	return req, nil
+}
+
+func getQueriesByDashboard(api, dashboardUid, httpMethod string, queryList *[]query) ([]string, error) {
+	queries := []string{}
+	url := grafanaAddr + api + dashboardUid
+	req, err := createNewHttpRequest(httpMethod, url, nil)
+	if err != nil {
+		return queries, fmt.Errorf("Error when creating http request: %v", err)
```

Comment:
```suggestion
		return queries, fmt.Errorf("error when creating http request: %v", err)
```
I think we don't capitalize error string?

Reply:
yes, my bad. thanks for pointing it out